redash/utils/configuration.py

```diff
@@ -55,7 +55,7 @@ def to_json(self):
         return json_dumps(self._config, sort_keys=True)
 
     def iteritems(self):
-        return self._config.iteritems()
+        return iter(self._config.items())
```

Comment:
`self._config.items()` is already an iterator since it's a dict view object, so I think it may not needed.

Reply:
Right, done.